content/docs/concepts/staking/_index.md

```diff
@@ -4,42 +4,46 @@ title: "Staking "
+Stake in an Autonity network is represented as the [Newton](/concepts/protocol-assets/newton/) stake token. Any network participant holding Newton becomes a [stakeholder](/glossary/#stakeholder) and is able to participate in securing the network and earn a share of [staking rewards](/glossary/#staking-rewards) proportionate to their stake in return. As default, Newton is in an unbonded and unlocked state and can be transferred to other stakeholders. On bonding, Newton is locked and no longer transferrable; Liquid Newton is minted for [delegated](/glossary/#delegated) stake in equal proportion to the stake locked. In this liquid staking model the Liquid Newton receives the staking reward entitlements due to the bonded stake it represents; Liquid Newton is transferrable. To redeem stake, the converse to bonding occurs. Liquid Newton for [delegated](/glossary/#delegated) stake is burned (so it is no longer tradable) and after the expiry of a [locking (unbonding) period](/concepts/staking/#unbondingperiod) the bonded Newton is redeemed.
 
-Staking is open - any network participant is able to purchase stake token and [delegate](/glossary/#delegate) stake to validators. 
+Staking is open - any network participant is able to purchase stake token and bond stake by [delegation](/glossary/#delegation) to validators. 
 
-Staking rewards are distributed to stake that is actively backing consensus. That is, to the subset of validator nodes participating in the [consensus committee](/glossary/#consensus-committee). Stake delegators to committee member validators receive a share of those rewards in proportion _pro rata_ to their share of the stake bonded to the committee.
+Staking rewards are distributed to [delegated](/glossary/#delegated) stake that is actively backing consensus. That is, to the subset of validator nodes participating in the [consensus committee](/glossary/#consensus-committee). Stake delegators to committee member validators receive a share of those rewards in proportion _pro rata_ to their share of the stake bonded to the committee.
 
+{{% alert title="Note" %}}
+Note that in Autonity's [Penalty-Absorbing Stake (PAS)](/concepts/staking/#penalty-absorbing-stake-pas) model, validator [self-bonded](/glossary/#self-bonded) stake does _not_ result in minting of liquid newton or earn staking rewards. Validator revenue is derived from commission, block proposal, and slashing rewards. See [validator economics](/concepts/validator/#validator-economics).
```

Comment:
> stake does _not_ result in minting of liquid newton or earn staking rewards. Validator revenue is derived from commission, block proposal, and slashing rewards

This bit is incorrect, see the `_selfReward` variable here https://github.com/autonity/autonity/blob/8183c96276bfc0764ad59b31cde423d6245329c0/autonity/solidity/contracts/Autonity.sol#L931

Reply:
Yes. Corrected to:

{{% alert title="Note" %}}
Note that in Autonity's [Penalty-Absorbing Stake (PAS)](/concepts/staking/#penalty-absorbing-stake-pas) model, validator [self-bonded](/glossary/#self-bonded) stake does _not_ result in minting of liquid newton. Validator revenue is derived from commission, block proposal, staking rewards on self-bonded stake, and slashing rewards. See [validator economics](/concepts/validator/#validator-economics).
{{% /alert %}}

Reply:
Commit 2401d40bc56baeb2145b5857c022b9a41d88eefb